src/generator.rs

```diff
@@ -29,7 +32,7 @@ pub const BUILD_DIRECTORY: &str = "proto/build";
 )]
 #[serde(rename_all = "kebab-case")]
 pub enum Language {
-    Rust,
+    Python,
```

Comment:
What if someone consuming the library wants to use it to generate Rust code (not for their own package, but as part of some composing tool)? Shouldn't we allow that too?

Reply:
We do, this is language enum for protoc languages we support